### PR TITLE
[NET-546] Move ee code to ee package, unify ee status and terminology

### PR DESCRIPTION
**Pull request by gabrielseibel1**
## Describe your changes

Moved all pieces of code that where specific to EE/Pro to correct directory.

Used variable functions that are filled in with pro code on pro initialization.

Main areas affected are metrics, relays, and ext. client acls.

Lot of the refactoring was done with automation/IDE tools, and go vet, staticcheck and go test pass, but nonetheless we should manually test this thoroughly since the diff is huge... lots of moving this around but logic should be the same.

Consolidated naming for paid/professional/enterprise version as "pro". Notes:
- Changes image tags
- Changes build tags
- Changes package names
- Doesn't change links to docs that mention "ee"
- Doesn't change parameters sent to PostHog that mention "ee"

## Provide testing steps

Test that pro features mentioned above are working and not available on community

### CODE REVIEW

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: ee/initialize.go
Comment (abhishek9686) [inline ee/initialize.go:1]: let's stick to only one terminology
should change EE to PRO everywhere in the code

**Review by abhishek9686 — CHANGES_REQUESTED** [review]

File: ee/initialize.go
Comment (gabrielseibel1) [reply ee/initialize.go:1]: https://github.com/gravitl/netmaker/pull/2538/commits/6f6c2aa6e230c54a93b1a70cd51961f3bbd1cfd2

**Review by abhishek9686 — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Review by abhishek9686 — CHANGES_REQUESTED** [review]
@gabrielseibel1 can you revert changes of ee -> pro for docker images and workflow, tags

File: pro/initialize.go [hunk 1]
```diff
@@ -1,33 +1,33 @@
 //go:build ee
```
Comment (Aceix) [inline pro/initialize.go:1]: may need to change this to pro as well since the build tags are changing

**Review by Aceix — CHANGES_REQUESTED** [review]

File: pro/initialize.go (hunk 1 above)
Comment (Aceix) [reply pro/initialize.go:1]: oh.. i now see the revert. might also want to check changes in .github/workflows/test.yml

Comment (gabrielseibel1) [reply pro/initialize.go:1]: thx!

**Review by Aceix — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Review by abhishek9686 — APPROVED** [review]

**Pull request merged**